src/Arcus.Scripting.Security/Arcus.Scripting.Security.psm1

```diff
@@ -34,4 +34,34 @@ function Get-AzCachedAccessToken {
     . $PSScriptRoot\Scripts\Get-AzCachedAccessToken.ps1
 }
 
-Export-ModuleMember -Function Get-AzCachedAccessToken
\ No newline at end of file
+Export-ModuleMember -Function Get-AzCachedAccessToken
+
+<#
+ .Synopsis
+  Grant a resource access to all resources within a resource group.
+
+ .Description
+  Grant a resource access to all resources present within a specific resource group.
+
+ .Parameter TargetResourceGroupName
+  The name of the resource group to which access should be granted.
+
+ .Parameter ResourceName
+  The name of the resource which should be granted access.
+
+ .Parameter ResourceGroupName
+  The name of the resource group where the resource is located which should be granted access.
```

Comment:
Can you switch position with these 2 parameters -> ResourceGroupName, ResourceGroup

Reply:
Yes, of course, they seem to be switched in the process.

Reply:
Done that. Resolving.